programs/programs/trua/calculator.py

```diff
@@ -0,0 +1,37 @@
+from programs.programs.calc import Eligibility, ProgramCalculator
+import programs.programs.messages as messages
+
+
+class Trua(ProgramCalculator):
+    income_limits = {
+        1: 66_300,
+        2: 75_750,
+        3: 85_200,
+        4: 94_560,
+        5: 102_250,
+        6: 109_800,
+        7: 117_400,
+        8: 124_950,
+    }
+
+    def eligible(self) -> Eligibility:
+        e = Eligibility()
+
+        # Income test
+        gross_income = int(self.screen.calc_gross_income("monthly", ["all"]))
+        income_band = int(Trua.income_bands[self.screen.household_size]/12)
+        
+        # Location test
+        location = self.screen.county
+
+        e.condition(location == "Denver County",
+                        messages.location())
```

Comment:
Our partner does not provide a county, but they do provide a zip code which we use to determine if the person is in an eligible county. Could you implement the county condition like it is for MySpark.

FYI, that is why I added `zipcode` to the dependencies, and not `county`